.github/workflows/build.yml

```diff
@@ -17,10 +17,36 @@ jobs:
       with:
         go-version: 1.20.3
 
-    - name: Format
+    - name: Set up protoc
       run: |
-        make format
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v25.0/protoc-25.0-linux-x86_64.zip
+        unzip protoc-25.0-linux-x86_64.zip
+        sudo mv bin/protoc /usr/local/bin/protoc
+        sudo mv include/* /usr/local/include/
+
+    - name: Install protoc-gen-go
+      run: go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1
+
+    - name: Generated protofiles are upto date
+      run: |
+        make generate-golang
         git diff --exit-code
```

Comment:
What's the point of running this? Just for the logs?

Reply:
to ensure proto changes are kept in sync with generated files

Reply:
updated. just keeping `git diff --exit-code` for now